### DISCUSSION TITLE
Taking datasets values to chart options

### DISCUSSION THREAD

**Discussion by cesarPano** [apache/superset#21475] [Ideas]
---
name: Taking datasets values to chart options
about: Taking datasets values to chart options 
labels: "#enhancement"
---
Hi.

After looking the issues, (many of them), I can´t find the how-to set chart options from a dataset value.
Doing this way in bullet chart Markers, Marker lines and Ranges would help in control the final look of the chart.
Maybe this could be useful in other charts.
Is there a way of getting this?

Thanks in advance.

**Comment by rusackas** [comment]
If I'm understanding your question/suggestion correctly, is it that you want to take chart customization options and add the ability to make them dynamic based on values? If so, that's all possible... we would just need clarity on exactly what sort of values/inputs you would expect for exactly which control(s). This sounds like a feature request rather than a bug, so I'll move it to a discussion for the time being. Let me know if I've misconstrued.

**Reply by cesarPano** [reply]
Hi. You have understood properly my idea. Maybe it´s me that set the issue in the wrong place.

What I want -> create a dashboard with many values (1 value -> 1 chart). These values are paired like partial-goal, so Bullet chart would be perfect for it if I could set a filter for the 'partial' data in 'METRIC', but later set another filter for the 'goal' in 'MARKERS'. 

A little more info ...

Data table sales:

id    name        goal       partial
1     cars          1000       350
2     bikes        9000       2300
3     houses     20           11
...

From here, I would like to get n Bullet charts with 
             'METRIC' 350 'MARKERS' 1000
             'METRIC' 2300 'MARKERS' 9000             
             'METRIC' 11 'MARKERS' 20
             ...
             
Such a customization could be useful for this chart and others.         
Thanks for your effort.
Regards.
